plugins/modules/proxmox.py

```diff
@@ -106,6 +106,14 @@
     description:
       - sets DNS search domain for a container
     type: str
+  tags:
+    description:
+      - List of tags to apply to the container.
+      - Tags must start with C([a-z0-9_]) followed by zero or more of the following characters C([a-z0-9_-+.]).
+      - Tags are only available in Proxmox 7+.
+    type: list
+    elements: str
+    version_added: 6.3.0
```

Comment:
The next expected release is 6.2.0 beginning of January.
```suggestion
    version_added: 6.2.0
```